src/libraries/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_Compile.cs

```diff
@@ -149,5 +151,866 @@ public void TFS_470021()
+           xmlns:xs='http://www.w3.org/2001/XMLSchema'>
+    <xs:simpleType name='foo'>
+        <xs:restriction base='xs:integer'>
+            <xs:minExclusive value='2' fixed='true'/>
+        </xs:restriction>
+    </xs:simpleType>
+    <xs:simpleType name='bar'>
+        <xs:restriction base='foo'>
+            <xs:minExclusive value='1'/>
+        </xs:restriction>
+    </xs:simpleType>
+</xs:schema>
+"
+                    },
+                    new object[]
+                    {  // minInclusive, derived type has larger value
+                        @"<?xml version='1.0' encoding='utf-8' ?>
+<xs:schema elementFormDefault='qualified'
+           xmlns:xs='http://www.w3.org/2001/XMLSchema'>
+    <xs:simpleType name='foo'>
+        <xs:restriction base='xs:integer'>
+            <xs:minInclusive value='3' fixed='true'/>
+        </xs:restriction>
+    </xs:simpleType>
+    <xs:simpleType name='bar'>
+        <xs:restriction base='foo'>
+            <xs:minInclusive value='4'/>
+        </xs:restriction>
+    </xs:simpleType>
+</xs:schema>
+"
+                    },
+                    new object[]
+                    {  // minInclusive, derived type has smaller value
+                        @"<?xml version='1.0' encoding='utf-8' ?>
+<xs:schema elementFormDefault='qualified'
+           xmlns:xs='http://www.w3.org/2001/XMLSchema'>
+    <xs:simpleType name='foo'>
+        <xs:restriction base='xs:integer'>
+            <xs:minInclusive value='3' fixed='true'/>
+        </xs:restriction>
+    </xs:simpleType>
+    <xs:simpleType name='bar'>
+        <xs:restriction base='foo'>
+            <xs:minInclusive value='2'/>
+        </xs:restriction>
+    </xs:simpleType>
+</xs:schema>
+"
+                    }
+                    // NOTE: the cases of totalDigits, with larger and smaller
+                    // values are not tested here as Issue 34426 was found
+                    // and will be addressed separately.
+                    // NOTE: the case of fractionDigits with a larger
+                    //   value is not tested here as Issue 34413 was found,
+                    //   and will be addressed separately.
+                    // NOTE: the case of fractionDigits with a smaller
+                    //   value is not tested here as Issue 34418 was found,
+                    //   and will be addressed separately.
+                };
+            }
+        }
+
+        [Theory]
+        [MemberData(nameof(FacetBaseFixed_Throws_TestData))]
+        public void FacetBaseFixed_Throws(string schema)
+        {
+            XmlSchemaSet ss = new XmlSchemaSet();
+            ss.Add(null, XmlReader.Create(new StringReader(schema)));
+
+            Exception ex = Assert.Throws<XmlSchemaException>(() => ss.Compile());
+            Assert.Contains("fixed='true'", ex.Message);
+
+            // Issue 30218: invalid formatters
+            Regex rx = new Regex(@"\{.*[a-zA-Z ]+.*\}");
+            Assert.Empty(rx.Matches(ex.Message));
+
```

Comment:
NIT: empty line, and please remove invalid formatters assert, they wouldn't be put back

Reply:
All the Regex's have been removed.  There is no longer an empty line between the last line of code in the method and the closing curly brace.